100MW_FPPwGlobalSupply.xml

```diff
@@ -0,0 +1,784 @@
+        </Source>
+    </config>
+  </facility>
+
+   <facility>
+    <name>Bruce A1</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.016467</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce A2</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.016467</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce A3</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.01625</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce A4</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.01625</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce B5</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.017702</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce B6</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.017702</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce B7</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.017702</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce B8</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.017702</throughput>
+        </Source>
+    </config>
+  </facility>
+
+  <!--(PI) Stands for Post Idle-->
+    <facility>
+    <name>Bruce A3 (PI)</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.01625</throughput>
+        </Source>
+    </config>
+  </facility>
+
+    <facility>
+    <name>Bruce A4 (PI)</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.01625</throughput>
+        </Source>
+    </config>
+  </facility>
+
+  <!--(R) Stands for Refurbished-->
+  <facility>
+    <name>Bruce A1 (R)</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>0.016467</throughput>
+        </Source>
+    </config>
+  </facility>
```

Comment:
We should discuss whether it is valuable enough to have each of these named separately (maybe for debugging), or whether we can define generic CANDUs at the appropriate throughputs and then just start them in the appropriate year.